### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by codingUniv**

### CODE REVIEW

File: hardware/include/cs_io_if.vh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+    //control-status i/o (do not remove indentation)
```
Comment (jjts) [inline hardware/include/cs_io_if.vh:1]: File not needed. Signals shoul be at top

File: hardware/include/gen_if.vh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+   //General Interface Signals (do not remove indentation)
```
Comment (jjts) [inline hardware/include/gen_if.vh:1]: This file already exists in iob lib

File: hardware/include/iob_m_if.vh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+    //Master i/f (do not remove indentation)
```
Comment (jjts) [inline hardware/include/iob_m_if.vh:1]: File exists

File: hardware/include/iob_s_if.vh [hunk 4]
```diff
@@ -0,0 +1,8 @@
+    //Slave i/f - Native (do not remove indentation)
```
Comment (jjts) [inline hardware/include/iob_s_if.vh:1]: File exists

**Review by jjts — COMMENTED** [review]
See comments

File: hardware/include/gen_if.vh (hunk 2 above)
Comment (codingUniv) [reply hardware/include/gen_if.vh:1]: The reset signal doesn't have the same name (reset in iob-cache and rst in iob-lib)

File: hardware/include/iob_m_if.vh (hunk 3 above)
Comment (codingUniv) [reply hardware/include/iob_m_if.vh:1]: The parameters used to configure the width are not the same

File: hardware/include/iob_s_if.vh (hunk 4 above)
Comment (codingUniv) [reply hardware/include/iob_s_if.vh:1]: Signals and parameters names are not the same

File: hardware/include/cs_io_if.vh (hunk 1 above)
Comment (codingUniv) [reply hardware/include/cs_io_if.vh:1]: Should these signals be removed from the top's interface signals?

**Comment by codingUniv** [comment]
Another suggestion will be shared via another PR.

**Pull request closed without merging**